test/edge/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(TORCH_ROOT ${CMAKE_CURRENT_LIST_DIR}/../..)
+set(TEST_ROOT ${TORCH_ROOT}/test/edge)
+set(OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
+
+# Generate unboxing kernels
+set(GEN_COMMAND
+        "${PYTHON_EXECUTABLE}" -m torchgen.gen_executorch
+        --source-path=${TEST_ROOT}
+        --install_dir=${OUTPUT_DIRECTORY}
+        --tags-path=${TORCH_ROOT}/aten/src/ATen/native/tags.yaml
+        --aten_yaml_path=${TORCH_ROOT}/aten/src/ATen/native/native_functions.yaml
+        --use_aten_lib
+        --op_selection_yaml_path=${TEST_ROOT}/selected_operators.yaml
+        )
+set(GEN_COMMAND_sources
+        ${OUTPUT_DIRECTORY}/RegisterCodegenUnboxedKernelsEverything.cpp
+        ${OUTPUT_DIRECTORY}/Functions.h
+        ${OUTPUT_DIRECTORY}/NativeFunctions.h
+        )
+message(STATUS "Generating sources for unboxing kernels ${GEN_COMMAND}")
+add_custom_command(
+        COMMENT "Generating sources"
+        OUTPUT ${GEN_COMMAND_sources}
+        COMMAND ${GEN_COMMAND}
+        DEPENDS
+        ${TORCH_ROOT}/aten/src/ATen/native/native_functions.yaml
+        ${TORCH_ROOT}/aten/src/ATen/native/tags.yaml
+        ${TEST_ROOT}/templates/Functions.h
+        ${TEST_ROOT}/templates/NativeFunctions.h
+        ${TEST_ROOT}/templates/RegisterCodegenUnboxedKernels.cpp
```

Comment:
you gotta add the torchgen py files as dependencies too, or it won't rebuild when python changesq